apex.js/index.d.ts

```diff
@@ -0,0 +1,8 @@
+// Type definitions for apex.js 2.0
+// Project: https://github.com/apex/node-apex
+// Definitions by: Yoriki Yamaguchi <https://github.com/y13i>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference types="aws-lambda" />
+
+export default function λ(fn: (event?: any, context?: AWSLambda.Context) => any): (event: any, context: AWSLambda.Context, callback: AWSLambda.Callback) => void;
```

Comment:
If `event` and `context` are reliably provided, they should not be optional parameters. So use `fn: (event: any, context: AWSLambda.Context) => any`